client/src/main/java/au/org/consumerdatastandards/client/api/BankingProductsAPI.java

```diff
@@ -589,6 +590,7 @@ public Type resolve(Response response) {
             Integer versionNumber = Integer.parseInt(version);
             switch (versionNumber) {
                 case 2:
+                case 3:
```

Comment:
This handling of v3 of BankingProduct is virtual - I ended up NOT creating BankingProductV3 as there are no differences with V2.